enos/k8s/enos-scenario-k8s.hcl

```diff
@@ -0,0 +1,147 @@
+scenario "k8s" {
+  matrix {
+    edition = ["oss", "ent"]
+  }
+
+  terraform_cli = terraform_cli.default
+  terraform     = terraform.k8s
+
+  providers = [
+    provider.enos.k8s,
+    provider.helm.default,
+  ]
+
+  locals {
+    image_path = abspath(var.vault_docker_image_archive)
+
+    image_repo = var.vault_image_repository != null ? var.vault_image_repository : matrix.edition == "oss" ? "hashicorp/vault" : "hashicorp/vault-enterprise"
+    image_tag = matrix.edition == "oss" ? var.vault_product_version : "${var.vault_product_version}-ent"
```

Comment:
The `var.vault_product_version` for `ent` does append `+ent` to version for enterprise editions

Reply:
This is for the image tag and unfortunately they break the pattern and use `-ent` in the tag. This might be due to the fact that `+` is not supported as a legal character for a docker tag.